src/components/button/button.js

```diff
@@ -0,0 +1,8 @@
+import button from './button.tmpl';
+import Handlebars from 'handlebars';
+import './button.less';
+
+const template = Handlebars.compile(button);
+
+export default (buttonId, buttonText) =>
+	template({ buttonId: buttonId, buttonText: buttonText });
```

Comment:
```suggestion
	template({ buttonId, buttonText });
```

Если название свойства и переменной совпадают, то можно использовать короткие свойства

Reply:
Логично

Reply:
подправил